actioncable/lib/action_cable/remote_connections.rb

```diff
@@ -45,6 +45,7 @@ def disconnect
         end
 
         # Returns all the identifiers that were applied to this connection.
+        remove_method :identifiers
```

Comment:
It doesn't make sense to remove a method that we defined ourselves. Can we just change the `class_attribute` definition to not define the instance method?